src/builtin.cpp

```diff
@@ -516,6 +534,10 @@ static int builtin_bind(parser_t &parser, io_streams_t &streams, wchar_t **argv)
                 sets_bind_mode = w.woptarg;
                 break;
             }
+            case 'L': {
```

Comment:
This is fine. I'm just making a meta comment to get people thinking that we really should establish a pattern for dealing with mutually exclusive arguments that should really result in an error rather than seemingly random behavior.